api/agentpb/agent.proto

```diff
@@ -133,6 +133,12 @@ message StartActionRequest {
     string dsn = 1;
     string table = 2;
   }
+  // MongoDBExplainParams describes MongoDB EXPLAIN action parameters.
+  message MongoDBExplainParams {
+    string dsn = 1;
+    string database = 2;
+    string query = 3;
+  }
```

Comment:
Is the json is the only format that we are going to implement?
If not maybe we should add output_format here?

Reply:
I thought about that but in contrast to MySQL, MongoDB only handle JSONs.
I don't think we want to implement a custom format.